docs/_static/custom.css

```diff
@@ -55,7 +55,9 @@ code.xref.docutils.literal {
 }
 
 div.viewcode-block:target {
-    background: inherit;
+    background:#dadada;
```

Comment:
Other elements defined in this stylesheet seem to be specific in form of `background-color` as opposed to `background`, should we follow the same?

Reply:
Good point. Actually `background` inherits many different background attributes, so I added the inherit back in (although it's not really inheriting anything, but could in the future) for background and set background-color explicitly.